services/aur/aur.service.js

```diff
@@ -54,29 +56,25 @@ class AurLicense extends BaseAurService {
     {
       title: 'AUR license',
       namedParams: { packageName: 'android-studio' },
-      staticPreview: this.render({ license: 'Apache' }),
+      staticPreview: renderLicenseBadge({ license: 'Apache' }),
     },
   ]
 
   static defaultBadgeData = { label: 'license' }
 
-  static render({ license }) {
-    return { message: license, color: 'blue' }
-  }
-
   transform(json) {
-    const license = json.results.License
-    if (!license) {
+    const licenses = json.results[0].License
+    if (!licenses) {
       throw new NotFound({ prettyMessage: 'not specified' })
     }
 
-    return { license }
+    return { licenses }
   }
 
   async handle({ packageName }) {
     const json = await this.fetch({ packageName })
-    const { license } = this.transform(json)
-    return this.constructor.render({ license })
+    const { licenses } = this.transform(json)
+    return renderLicenseBadge({ licenses, color: 'blue' })
```

Comment:
I opted to extend our license badge helper to let call sites specify a default color to use instead of the dynamically derived colors based on the license values. This seems a reasonable change in its own right, though opted to do so here since we have historically hardcoded the color for this badge and didn't want to introduce color changes as part of updating to the latest API